code/modules/projectiles/ammunition/energy/laser.dm

```diff
@@ -6,7 +6,7 @@
 /obj/item/ammo_casing/energy/laser/hellfire
 	projectile_type = /obj/projectile/beam/laser/hellfire
 	e_cost = 100
-	select_name = "maim"
+	select_name = "Ignite"
```

Comment:
why capitalized? none of the others are

Reply:
Got it. You are right about that, fixing it up.